ansible/README.md

```diff
@@ -17,75 +17,21 @@ For more explanation on using Ansible in general, you may want to read:
-   Install upstream Docker
- - openmicroscopy.haproxy
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-haproxy)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/haproxy/)):
-   HAProxy installation and configuration
- - openmicroscopy.ice
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-ice)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/ice/)):
-   Install ZeroC Ice
- - openmicroscopy.java
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-java)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/java/)):
-   Install a given JRE or JDK
- - openmicroscopy.logrotate
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-logrotate)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/logrotate/)):
-   Customise log-rotation
- - openmicroscopy.lvm-partition
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-lvm-partition)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/lvm-partition/)):
-   Create a formatted LVM volume
- - openmicroscopy.munin
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-munin)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/munin/)):
-   Setup a Munin monitoring server
- - openmicroscopy.nfs-mount
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-nfs-mount)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/nfs-mount/)):
-   Manage NFS4 mounts
- - openmicroscopy.nginx
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-nginx)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/nginx/)):
-   Install upstream nginx
- - openmicroscopy.postgresql
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-nginx)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/nginx/)):
-   Install upstream PostgreSQL
- - openmicroscopy.redis
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-nginx)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/nginx/)):
-   Install redis
- - openmicroscopy.sudoers
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-nginx)/[Galaxy](https://galaxy.ansible.com/openmicroscopy/nginx/)):
-   Configure sudoers
-
-Other core roles can be found under the [openmicroscopy](https://galaxy.ansible.com/openmicroscopy/) account on Galaxy.
-
-
-### on GitHub ###
-
-Roles which are not available on Galaxy should be considered in development. For example:
-
- - hosts-populate
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-hosts-populate)):
-   Populates /etc/hosts with static addresses for a list of hostgroups
- - omero-user
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-omero-user)):
-   Create OMERO user accounts and groups
- - omero-web-apps
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-omero-web-apps)):
-   Install OMERO.web plugins
- - reboot-server
-   ([GitHub](https://github.com/openmicroscopy/ansible-role-reboot-server)):
-   Reboot a server, optionally wait for it to return.
+Core roles can be found under the
```

Comment:
See https://github.com/sbesson/infrastructure/tree/roles_cleanup/ansible for the rendering of the last commit  6b04d4f